### PR TITLE
@alloy => Made Fastfile more awesome.

### PR DESCRIPTION
**Pull request by ashfurrow**
Less `sh` and more using build-in actions. 

Was just waiting on https://github.com/KrauseFx/fastlane/pull/175

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -98,7 +100,10 @@ lane :deploy do
   )
 
   # Tag release and push to GitHub
-  sh "cd .. ; git tag #{version_number} ; git push --tags"
+  add_git_tag ({
```
Comment (orta) [inline fastlane/Fastfile]: @alloy can this be code-golf'd?

Comment (ashfurrow) [inline fastlane/Fastfile]: What does that mean?

Comment (orta) [inline fastlane/Fastfile]:
```
$ pry

[1] pry(main)> def function(a) puts a end
=> :function
[4] pry(main)> function({})
{}
=> nil
[6] pry(main)> version_number = 1
=> 1
[7] pry(main)> function({ tag: version_number })
{:tag=>1}
=> nil
[8] pry(main)> function(tag: version_number)
{:tag=>1}
=> nil
[9] pry(main)> function tag:version_number
{:tag=>1}
=> nil
[10] pry(main)>
```

**Comment by ashfurrow** [comment]
Also code-golf'd.

**Pull request merged**